### PR TITLE
Add ability to change sw ip to lockkeeper

### PR DESCRIPTION
**Pull request by rtretyak**

### CODE REVIEW

File: src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/MultiFloodlightsSpec.groovy [hunk 1]
```diff
@@ -107,4 +114,32 @@ class MultiFloodlightsSpec extends HealthCheckSpecification {
         cleanupActions.each { it() }
         wait(WAIT_OFFSET) { northbound.getAllSwitches().each { assert it.state == ACTIVATED } }
     }
+
+    def "System supports case when switch uses different networks to connect to FLs, i.e. has diff ips"() {
```
[the remark this answers is not in the document]
Comment (andriidovhan) [reply src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/MultiFloodlightsSpec.groovy:118]: `@Tags([LOCKKEEPER])` ?

File: src-java/testing/test-library/src/main/java/org/openkilda/testing/service/lockkeeper/LockKeeperServiceImpl.java [hunk 2]
```diff
@@ -227,6 +228,19 @@ public void reviveFloodlight(String region) {
                 getPort(kafkaBootstrapServer)));
     }
 
+    @Override
+    public void changeSwIp(Switch sw, String region, String oldIp, String newIp) {
+        lockKeepersByRegion.get(region).exchange("/floodlight/nat/input", HttpMethod.POST,
```
[the remark this answers is not in the document]
Comment (rtretyak) [reply src-java/testing/test-library/src/main/java/org/openkilda/testing/service/lockkeeper/LockKeeperServiceImpl.java:233]: Added to comply with other methods, but tbh I'd rather removed it from all the methods at all

**Comment by rtretyak** [comment]
All review comments have been taken into account

File: src-java/projectfloodlight/floodlight/build.gradle [hunk 3]
```diff
@@ -42,11 +42,11 @@ task cloneFloodlightRepo {
             logger.warn("Failed to fetch floodlight from git: {}", e.getMessage())
         }
     } else {
-        def repoUri = 'https://github.com/kilda/floodlight.git'
-        if (project.hasProperty("floodlightGitRepo")) {
-            repoUri = project.property("floodlightGitRepo")
-        }
-        def grgit = Grgit.clone(dir: repo.absolutePath, uri: repoUri, refToCheckout: versionTag)
+        def repoUri = 'https://github.com/rtretyak/floodlight.git'
```
[the remark this answers is not in the document]
Comment (rtretyak) [reply src-java/projectfloodlight/floodlight/build.gradle:45]: was a temporary change to test the floodlight fix https://github.com/kilda/floodlight/pull/10
reverting it back

**Review by andriidovhan — APPROVED** [review]

**Pull request merged**